src/angular/components/view.component.ts

```diff
@@ -201,6 +201,14 @@ export class ViewComponent implements OnDestroy, OnInit {
         }
     }
 
+    toggleFieldCount(field: FieldView) {
+        const f = (field as any);
+        f.showCount = !f.showCount;
+        if (f.showCount) {
```

Comment:
This `if` block can be removed in that we don't need to collect this particular event, unless showing the count takes the state from hidden to visible, otherwise this shouldn't be necessary (and if it is necessary, please re-use the same event for toggling a hidden field view action).